modules/virt-copying-the-ssh-command.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-accessing-vm-consoles.adoc
+
+:_content-type: PROCEDURE
+[id="virt-copying-the-ssh-command_{context}"]
+= Copying the SSH Command from the web console
+
+Copy the command to access a running virtual machine (VM) via SSH.
+
+.Procedure
+
+. In the {product-title} console, click *Virtualization* -> *VirtualMachines* from the side menu.
+. Use the search and filters to find your VM in the list.
+. From the *Actions* list, select *Copy SSH Command*. You can now connect to your VM via SSH by using the command in the OpenShift CLI (`oc`).
```

Comment:
s/the command/this command (sounds better in my head when I read that sentence)

Reply:
Sabrina,
I'm not a peer reviewer, so I hope you don't mind my suggestion. :-) I got CCed automatically, which is how I happened to see Shikha's comment. I think that "you can now connect" and "by using the command" might be problematic for localization. I recommend using concrete verbs whenever possible. 
Also, if the user does not log in first, the command won't work, so you might want to add it as a prerequisite.

Suggestion:

.Prerequisites
* You must log in to your cluster by using the OpenShift CLI (`oc`).
...
. From the *Actions* list, select *Copy SSH Command*. The SSH command is copied to your clipboard.
. Paste (Enter?) the command on the CLI...

Reply:
Re: "Copy SSH Command". It's worth double-checking the capitalization. I had a quick look at a 4.11 cluster. Although that command doesn't appear there, the kebab menu items appear to be sentence case, not title case.